pkg/render/logstorage/linseed/linseed.go

```diff
@@ -310,6 +310,11 @@ func (l *linseed) linseedDeployment() *appsv1.Deployment {
 		envVars = append(envVars, corev1.EnvVar{Name: "LINSEED_EXPECTED_TENANT_ID", Value: l.cfg.Tenant.Spec.ID})
 		envVars = append(envVars, corev1.EnvVar{Name: "BACKEND", Value: "elastic-single-index"})
 		envVars = append(envVars, corev1.EnvVar{Name: "LINSEED_MULTI_CLUSTER_FORWARDING_ENDPOINT", Value: fmt.Sprintf("https://tigera-manager.%s.svc:9443", l.cfg.Tenant.Namespace)})
+		envVars = append(envVars, corev1.EnvVar{Name: "ELASTIC_DYNAMIC_INDEX_CREATION_ENABLED", Value: "true"})
```

Comment:
Why do we only set this if a tenant is provided?

Reply:
We should not set this.